keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -330,6 +329,9 @@ class ClusterHandler(
       return@coroutineScope tasks
     }
 
+  private fun createServerGroupOrcaJob(diff: ResourceDiff<ServerGroup>, refId: Int, resource: Resource<ClusterSpec>): Map<String, Any?> =
+    diff.createServerGroupJob(refId) + resource.spec.deployWith.toOrcaJobProperties("Amazon")
```

Comment:
Looks like there was already an extension function defined on the `ResourceDiff` object with almost the same purpose. Wouldn't it make sense to just add the additional parameters to that method instead of adding another level of indirection?

Reply:
@luispollo My concern with adding to the existing function is that I felt like it would violate the "does one thing" aspect of it. But I did end up changing the additional function so it's also an extension function.